config/initializers/filter_parameter_logging.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Be sure to restart your server when you modify this file.
```

Comment:
I'm not sure if this comment is necessary anymore. Rails dev should kind of now that changing anything in initializers should require a reload of the server in development.

Reply:
This has been autogenerated by Rails actually :sweat_smile:

Reply:
O interesting, is there an automated way to generate an initializer now?

Reply:
Not for any initializer no but when upgrading Rails, the update task does this automatically: https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#the-update-task

Reply:
O icic thank you for clarifying 👍